db/migrate/20221020222745_create_osco_warrants.rb

```diff
@@ -0,0 +1,17 @@
+class CreateOscoWarrants < ActiveRecord::Migration[6.0]
+  def change
+  
+    create_table :osco_warrants do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :middle_name
+      t.date :birth_date
+      t.string :case_number
+      t.decimal :bond_amount, precision: 14, scale: 2 
+      t.date :issued
+      t.string :counts
+
```

Comment:
Can you add a unique constraint here on the first_name, last_name, middle_name, birth_date, and case_number? 

I think this will work:
```suggestion
`add_index :people, [:first_name, :last_name, :middle_name, :birth_date, :case_number], unique: true`
```

Reply:
just saw this, yes I can!

Reply:
will add this after my review

Reply:
sorry I already added this